src/cr/cube/dimension.py

```diff
@@ -1300,7 +1300,7 @@ def reference_column_idx(self):
         """Optional int index of base column as reference for the pairwise sig tests."""
```

Comment:
docstring is out of date now, right? 
"Optional int index" -> "Optional element id" (which is either an int/str if you want to have the type)

(I think Mike's arguing in slack that this should be called the "code", but this would be the only place we use that terminology)